client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -100,11 +96,6 @@ export class ProductPurchaseFeaturesList extends Component {
 					liveChatButtonEventName={ 'calypso_livechat_my_plan_business' }
 				/>
 				<CustomDomain selectedSite={ selectedSite } hasDomainCredit={ planHasDomainCredit } />
-				<BusinessOnboarding
-					isWpcomPlan
```

Comment:
😕 Is this WPCOM concierge?

Reply:
Yeah, I shouldn't have removed this -- turns out this is the `getBusinessFeatures()` method which is used to show WP.com features (which are also handled in this file).

I've reverted the cards removing commits and filed new PRs, where this is fixed.